### PR TITLE
Fix footer aceburgess

### PR DESCRIPTION
**Pull request by aceburgess**
@afeld @sidwatal @orta

Added `clear` to footer CSS so that it would stay at the bottom of the page.

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -60,26 +59,49 @@ def update
     end
   end
 
- 
+
   private
 
+<<<<<<< HEAD
```
Comment (rickyc) [inline app/controllers/events_controller.rb]: Merge conflict.

**Comment by aceburgess** [comment]
I've updated the conflict. Thanks!

File: app/assets/stylesheets/stylesheet.css [hunk 2]
```diff
@@ -116,6 +116,7 @@ a:active {
 .cfooter{
 	text-align: center;
 	font-size: 1.3em;
+  clear: both;
```
Comment (rickyc) [inline app/assets/stylesheets/stylesheet.css]: :+1: with the 2 space indentation. Could you update the rest of the file to follow that convention?

**Comment by aceburgess** [comment]
Updated the rest of the spreadsheet file to have 2 space indentation instead of 4.

**Pull request merged**